notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -126,32 +126,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from openvino.runtime import Core\n",
+    "from openvino.runtime import serialize, Core\n",
     "\n",
     "ie = Core()\n",
-    "onnx_model_path = \"model/segmentation.onnx\"\n",
-    "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
```

Comment:
Why are these lines deleted?

Reply:
132 and 133 is rewritten to model = ie.read_model("model/segmentation.onnx")
I will include the step to show "How to compile the segmentation model "
Thanks for the feedback